src/main/java/com/starkinfra/IndividualDocument.java

```diff
@@ -0,0 +1,660 @@
+     * <p>
+     * Return:
+     * @return list of IndividualDocument objects with updated attributes
+     * @throws Exception error in the request
+     */
+    public static List<IndividualDocument> create(List<?> documents) throws Exception {
+        return IndividualDocument.create(documents, null);
+    }
+
+    /**
+     * Retrieve a specific IndividualDocument
+     * <p>
+     * Receive a single IndividualDocument object previously created in the Stark Infra API by passing its id
+     * <p>
+     * Parameters:
+     * @param id [string]: IndividualDocument unique id. ex: "5656565656565656"
+     * @param user [Organization/Project object, default null]: Organization or Project object. Not necessary if starkinfra.Settings.user was set before function call
+     * <p>
+     * Return:
+     * @return IndividualDocument object with updated attributes
+     * @throws Exception error in the request
+     */
+    public static IndividualDocument get(String id, User user) throws Exception {
+        return Rest.getId(data, id, user);
+    }
+
+    /**
+     * Retrieve a specific IndividualDocument
+     * <p>
+     * Receive a single IndividualDocument object previously created in the Stark Infra API by passing its id
+     * <p>
+     * Parameters:
+     * @param id [string]: IndividualDocument unique id. ex: "5656565656565656"
+     * <p>
+     * Return:
+     * @return IndividualDocument object with updated attributes
+     * @throws Exception error in the request
+     */
+    public static IndividualDocument get(String id) throws Exception {
+        return IndividualDocument.get(id, null);
+    }
+
+    /**
+     * Retrieve IndividualIdentities
+     * <p>
+     * Receive a generator of IndividualDocument objects previously created in the Stark Infra API.
+     * Use this function instead of page if you want to stream the objects without worrying about cursors and pagination.
+     * <p>
+     * Parameters:
+     * @param params map of parameters for the query
+     * limit [integer, default null]: maximum number of objects to be retrieved. Unlimited if null. ex: 35
+     * after [string, default null] date filter for objects created only after specified date. ex: "2020-03-10"
+     * before [string, default null] date filter for objects created only before specified date. ex: "2020-03-10"
+     * status [list of strings, default null]: filter for status of retrieved objects. Options: ["canceled", "created", "expired", "failed", "processing", "signed", "success"]
+     * tags [list of strings, default null]: tags to filter retrieved objects. ex: ["tony", "stark"]
+     * ids [list of strings, default null]: list of ids to filter retrieved objects. ex: ["5656565656565656", "4545454545454545"]
+     * @param user [Project object, default null]: Project object. Not necessary if StarkInfra.Settings.user was set before function call
```

Comment:
@param user [Organization/Project object, default null]: Organization or Project object. Not necessary if starkinfra.Settings.user was set before function call